src/main/java/duke/storage/LoadLineParser.java

```diff
@@ -107,10 +107,11 @@ public static User parseUser(String line) {
         int activityLevel = Integer.parseInt(splitLine[3]);
         int originalWeight = Integer.parseInt(splitLine[4]);
         String sex = splitLine[5];
+        String lastDate = splitLine[6];
         if (sex.equals("M")) {
-            return new User(name, age, height, Gender.MALE, activityLevel, originalWeight);
+            return new User(name, age, height, Gender.MALE, activityLevel, originalWeight, lastDate);
         } else {
-            return new User(name, age, height, Gender.FEMALE, activityLevel, originalWeight);
+            return new User(name, age, height, Gender.FEMALE, activityLevel, originalWeight, lastDate);
```

Comment:
You could refactor this a little. Suggestion:
<pre><code>
if (sex.equals("M")) {
    gender = Gender.Male;
} else {
    gender = gender.Female;
} 
return new User(name, age, height, gender, activityLevel, originalWeight, lastDate);
</code></pre>